README.md

```diff
@@ -1094,6 +1095,23 @@ params do
 end
 ```
 
+### Alias
+
+It's possible to set alias for parameters using `as` (it can be useful when refactoring an existing API):
```

Comment:
You can set an alias for parameters using `as`, which can be useful when refactoring existing APIs.